src/content/blog/lets-fund-stas-maintainer.md

```diff
@@ -0,0 +1,41 @@
+
+**Why is DDEV Maintenance Important?** You and DDEV are in engulfed in a maelstrom of change. Upstream technologies change on your weekly. Things break from all directions. If DDEV weren't cared for daily, it would fail to serve you within a pretty short time. 
+
+* PHP 8.4 support was added today to DDEV HEAD (along with versions 5.6-8.3!). 
+* Drupal 11 is coming out this week. DDEV has already had support for some time. 
+* PostgreSQL 16 and MariaDB 11.4 already have support, and MySQL 8.4 will shortly when some upstream support becomes available. 
+* Recently [MariaDB changed their `mysqldump` output file format](mariadb-dump-breaking-change.md), causing a few of you to pull your hair out, until DDEV incorporated informal and then fully-integrated workarounds, so there are some of you that don't even know this happened.
+
+I'm just naming a tiny few of the things DDEV has had to react to in the last few months.
+
+**What is our current financial situation?** Currently DDEV receives donations in the range of USD$3000-$5000/month. Our expenses are in that same range, the bulk being part-time hourly support for Stas. As of today we have about $18,000 in the bank. Financial reports are provided with every meeting of the [DDEV Advisory Group](https://github.com/orgs/ddev/discussions/categories/ddev-advisory-group).
+
+**What do we want to do? Increase income by about $3000/month.** In order for Stas to be able to leave behind his client work, we need approximately $3000/month more in committed support for the next year. This could be $1,000/month from three organizations (joining fabulous supporting agency [Tag1 Consulting](https://tag1consulting.com)), $500/month from 6 organizations, joining [i-gelb](https://i-gelb.net/). Or $100/month from 30 organizations, joining [so many individual and corporate sponsors](https://ddev.com/#supporters) who keep DDEV going.
+
+**How can you or your organization support DDEV?** For agencies, hosting folks, other organizations, we're happy to bill you monthly or annually, just ask! It's easy. Some orgs prefer to use GitHub Sponsors, that's fine too. For individuals, the easiest way is [GitHub Sponsors](https://github.com/sponsors/ddev)
+
+**Is your organization budgeting for 2025?** Please remember DDEV!
+
+**Get in touch!** I'd be happy to talk to you or your organization. Send [an email](mailto:randy.fay%40ddev.com). Make an [appointment for a video call](https://cal.com/randyfay/30min).
+
+We're so glad to be bringing you DDEV and working with an amazing community to keep things running! Thanks for all your contributions and for joining us in this journey!
```

Comment:
"to be bringing you DDEV" sounds off somehow. it is the correct form but still it sounds strange. would something like "We're so glad to provide DDEV to you and working with an amazing community to keep things running!" ?